tools/tfuser/main.go

```diff
@@ -104,34 +104,66 @@ func main() {
 						},
 						{
 							Name:  "add-node",
-							Usage: "add a node to a existing network",
+							Usage: "Add a node to a existing network",
 							Flags: []cli.Flag{
 								cli.StringFlag{
 									Name:  "node",
-									Usage: "node ID of the node to add to the network",
+									Usage: "Node ID of the node to add to the network",
 								},
 								cli.StringFlag{
 									Name:  "subnet",
-									Usage: "subnet to use on this node. The subnet needs to be included in the IP range of the network",
+									Usage: "Subnet to use on this node. The subnet needs to be included in the IP range of the network",
 								},
 								cli.UintFlag{
 									Name:  "port",
+									Usage: "Wireguard port to use. if not specified, tfuser will automatically check BCDB for free fort to use",
+								},
+								cli.UintFlag{
+									Name:  "l3",
 									Usage: "Wireguar port to use. if not specified, tfuser will automatically check BCDB for free fort to use",
 								},
+								cli.BoolFlag{
+									Name:  "force-hidden",
+									Usage: "Forcibly mark the node as hidden, even if it is registered with public endpoints",
+								},
 							},
 							Action: cmdsAddNode,
 						},
 						{
 							Name:  "remove-node",
-							Usage: "prints the wg-quick configuration file for a certain user in the network",
+							Usage: "Removes a Network Resource from the network",
 							Flags: []cli.Flag{
 								cli.StringFlag{
 									Name:  "node",
-									Usage: "node ID to remove from the network",
+									Usage: "Node ID to remove from the network",
 								},
 							},
 							Action: cmdsRemoveNode,
 						},
+						{
+							Name:   "graph",
+							Usage:  "create a dot graph of the network",
+							Action: cmdGraphNetwork,
+						},
+						{
+							Name:  "add-access",
+							Usage: "Add external access to the network",
+							Flags: []cli.Flag{
+								cli.StringFlag{
+									Name:  "node",
+									Usage: "Node ID of the node which will act as an access point",
+								},
+								cli.StringFlag{
+									Name:  "subnet",
+									Usage: "Local subnet which will have access to the network",
+								},
+								cli.BoolFlag{
```

Comment:
It sets a subnet for this endpoint, similar to how every net resource also has a subnet. A wireguard IP address is then derived for the endpoint (like it happens for network resources). The subnet is routable from every node in the network. E.G. if you say `--subnet 10.1.0.0/24`, you can set up a bridge on your laptop and attach containers to this bridge, and they will then be reachable from inside the network resources. I'm not really sure how this should be worded, open for suggestions

Reply:
From what I've understood of this flag, what is does is it make the endpoint of the wg connection to be an IPv4 instead of using the public ipv6 address of the access node.
None of this is explained.

Reply:
Oops, I thought this was talking about the subnet flag. My bad.